chatter.js

```diff
@@ -1,12 +1,80 @@
 Chatter = {
-    options: {
-        messageLimit: 30,
-        nickProperty: "username",
+  options: {
+    messageLimit: 30,
+    nickProperty: "username",
+  }
+};
+
+Chatter.configure = function (opts) {
+  _.extend(this.options, opts);
```

Comment:
Certain that `this` will be set to the right context, and not to the caller's context?